app/presenters/date_stamp_context_presenter.rb

```diff
@@ -13,18 +13,21 @@ def show?
 
   def first_name_changed?
     return false unless @applicant
+    return false if @date_stamp_context&.first_name.blank?
```

Comment:
Would it be easier to add to the `show?` method and check if any attributes are present?